assert/assertions_test.go

```diff
@@ -1369,6 +1370,25 @@ func TestDirExists(t *testing.T) {
 	True(t, DirExists(mockT, "../_codegen"))
 }
 
+func TestEqualFileContent(t *testing.T) {
```

Comment:
Could you also add a test demonstrating what would happen if the user pointed to a binary file?